R/epi_recipe.R

```diff
@@ -95,7 +95,7 @@ epi_recipe.epi_df <-
     keys <- key_colnames(x) # we know x is an epi_df
 
     var_info <- tibble(variable = vars)
-    key_roles <- c("geo_value", "time_value", rep("key", length(keys) - 2))
+    key_roles <- c("geo_value", rep("key", length(keys) - 2), "time_value")
```

Comment:
[This is updated](https://github.com/cmu-delphi/epiprocess/blob/187bd5d87eea3a4acc323a715d1ba32edb34cd49/R/epi_df.R#L188) in our upcoming [epiprocess PR](https://github.com/cmu-delphi/epiprocess/pull/477). If this line isn't changed then the wrong columns are associated with the wrong roles.

Reply:
I see. Is there a reason for that choice? I think the previous ordering was based on a Slack vote that @brookslogan took a while back? I can't seem to find it.

Reply:
I think [the Slack vote](https://delphi-org.slack.com/archives/C028AUR6MV1/p1674673612084949) asked for people's preference for the order to add arguments to a function, not how they want to order their epi_df.

Reply:
(Right, and this was about API data-fetching functions, rather than epiprocess ones.  In arrange_canonical I think we just borrowed this vote without doing another one... and I do wonder about my polling methodology anyway.)